encrypt_credentials.py

```diff
@@ -7,11 +7,12 @@
 import os
 from ansible.parsing.vault import VaultEditor, VaultSecret, is_encrypted
 from generate_example_from_schema import ExampleFileGenerator
+from builtins import str
 
 DEPLOYMENT_DIR = 'deployments'
 
 
-class VaultYaml(unicode):
+class VaultYaml(str):
```

Comment:
Does this still work in python2 if there is unicode in the credentials file?

Reply:
Yes this works for both python2 and 2. Builtins library is available in python 2.7 without any packages